assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/fields/BaseNumberInputField.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useCallback } from 'react';
+
+import { BaseFieldProps } from './types';
+import { NumberInput } from '@infraUI/inputs';
+
+const BaseNumberInputField: React.FC<BaseFieldProps> = ({
+	className,
+	disabled,
+	name,
+	format,
+	formatOnBlur,
+	parse,
+	placeholder,
+	getValue,
+	setValue,
+	value,
+}) => {
+	const fieldValue = Number(value ?? getValue());
+
+	const onChange = useCallback(
+		(value) => {
+			setValue(parse(value, name));
```

Comment:
`value` is already defined in upper scope

Reply:
thx, updated: https://github.com/eventespresso/event-espresso-core/pull/2888/commits/6e0783ab85a24ffcf09e1e3e8ab885d0c5115089